core/src/Util.js

```diff
@@ -205,12 +205,17 @@ export class Util {
         return Util.wrapBuffer(base64UrlToBuffer(key));
     }
 
-    static parseParamsFromUrl(url) {
+    static parseParamsFromUrl = url => {
         const query = url.split('?')[1];
         const result = {};
+        const supportedTypes = [];
         query.split('&').forEach(function(part) {
             const item = part.split('=');
-            result[item[0]] = decodeURIComponent(item[1]);
+            if (item[0] === 'supportedTransferDestinationType') { // for parsing set transfer destinations url callback parameters
```

Comment:
this method is for generic URL param parsing, there shouldn't be this kind of specific logic here. also why do we need this?